tests/network/localnet/conftest.py

```diff
@@ -353,12 +352,111 @@ def migrated_localnet_vm(
 
 @pytest.fixture(scope="module")
 def nncp_localnet_on_secondary_node_nic(
-    worker_node1: Node, nodes_available_nics: dict[str, list[str]], request: pytest.FixtureRequest
+    hosts_common_available_ports: list[str], request: pytest.FixtureRequest
 ) -> Generator[libnncp.NodeNetworkConfigurationPolicy]:
+    params = getattr(request, "param", {})
     mtu = None
-    if hasattr(request, "param"):
-        mtu = request.getfixturevalue(request.param)
+
+    mtu_fixture_name = params.get("mtu")
+    if mtu_fixture_name:
+        mtu = request.getfixturevalue(mtu_fixture_name)
```

Comment:
nit: This is very noisy, consider a one-liner:
```python
mtu = request.getfixturevalue(mtu_fixture_name) if (mtu_fixture_name := params.get("mtu")) else None
```

In a follow up, I would prefer to standardize the param that is passed, as a structured type.
I find the dict too scriptic and hard to sync them between callers and callee.
But we can do it in a follow up.